requirements.txt

```diff
@@ -1,5 +1,5 @@
 six
 tqdm
 torch>=0.4.0
-torchtext>=0.2.3
+git+https://github.com/pytorch/text
```

Comment:
We need a version here or we can host our own. Torchtext will break if we point to their github.